patches/extensions-common-permissions-api_permission.h.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/extensions/common/permissions/api_permission.h b/extensions/common/permissions/api_permission.h
+index 6354d5c66b254779a9a1a10816a9cf35a1df99b8..8630a5e5dca68c5cd6803136e5163c7edfbc09e1 100644
+--- a/extensions/common/permissions/api_permission.h
++++ b/extensions/common/permissions/api_permission.h
+@@ -272,6 +272,7 @@ class APIPermission {
+     kSearch = 228,
+     kTabGroups = 229,
+     kScripting = 230,
++    BRAVE_ID
```

Comment:
Do you know if we should leave room here for new Chromium permissions 231 and beyond?
I'm just wondering if this enum value is persisted to disk at all or not. 
I.e. if the permission ID changes later, (chromium adds a 231 and we change ours to 232) will that mess an already installed extension that has this permission?

Reply:
it works if I add some room, but there are couple of messages like:
1) https://source.chromium.org/chromium/chromium/src/+/master:extensions/common/permissions/api_permission.h;l=35;bpv=1;bpt=1?q=api_permission&ss=chromium
2) https://source.chromium.org/chromium/chromium/src/+/master:chrome/browser/extensions/extension_service.cc;l=1096;drc=effdbe12c7d315055a0a4ddfea3770f2dc2cf84d;bpv=1;bpt=1